chapters/en/_toctree.yml

```
@@ -44,6 +44,8 @@
     local: "unit2/cnns/mobilenetextra"
   - title: Resnet 
     local: "unit2/cnns/resnet"
+  - title: YOLO; You only Look once
```

I would suggest to put all the first letters in capital here:
```suggestion
  - title: YOLO; You Only Look Once
```